### PR TITLE
Updated landing page with changes on mockups

### PR DESCRIPTION
**Pull request by rleira**

### CODE REVIEW

File: public/modules/home/views/partials/footer.html [hunk 1]
```diff
@@ -1,89 +1,102 @@
...
+        <li><a href="#">Get Paid to Write</a></li>
+      </ul>
+    </div>
+
+    <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2">
+      <h3 class="footer-title">Connect</h3>
+
+      <ul>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">LinekIn</a></li>
```
Comment (arielron) [inline public/modules/home/views/partials/footer.html]: This should be Linkedin.

File: public/modules/home/views/home.client.view.html [hunk 2]
```diff
@@ -33,149 +18,112 @@
...
-                      </div>
+              <div class="intro col-sm-10">
+                  <h3>Get Started with Ignition</h3>
+                  <div class="col-sm-12">
+                      <p>To get you started, The Digital Garage has put together a base package for just $5 month.</p>
+                      <p>For $5 month, you get: 512 MB / 1 cpu, 10 GB of SSD Disk and 1TB of Transfer</p>
                   </div>
               </div>
           </div>
-          <div class="row">
```
Comment (arielron) [inline public/modules/home/views/home.client.view.html:121]: This two Items, Get Started with Ignition and the following are too close, one to another, please use margin top or separate them a bit more.

File: public/modules/home/views/partials/header.html [hunk 3]
```diff
@@ -7,7 +7,8 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/">{{ brand }}</a>
+      <a class="navbar-brand visible-xs" href="/">{{ brandSmall }}</a>
+      <a class="navbar-brand hidden-xs" href="/">{{ brand }}</a>
```
Comment (arielron) [inline public/modules/home/views/partials/header.html:11]: This appears to be bigger in the keynote snapshot, maybe increase the font one size or two.

File: public/modules/home/views/partials/footer.html [hunk 4]
```diff
@@ -1,89 +1,102 @@
...
-                    <textarea id="message" name="message" class="input-block-level footer-input" placeholder="Your Message" rows="5"></textarea>
-                </div>
-                <div class="controls">
-                    <button id="contact-submit" type="submit" class="btn btn-sky input-medium pull-right">Send</button>
-                </div>
-            </form>
-        </div>
+  <div class="row">
+    <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2 col-sm-offset-1">
+      <h3 class="footer-title">Product</h3>
```
Comment (arielron) [inline public/modules/home/views/partials/footer.html:4]: Maybe the headers should be vertically aligned with their child items and start at the same x coordinate.

File: public/dist/application.js [hunk 5]
```diff
@@ -860,13 +860,14 @@ angular.module('app.home').controller('HeaderController', ['$window', '$rootScop
   function ($window, $rootScope, $scope, $state, ngProgressFactory, $modal) {
 
     $scope.brand = 'The Digital Garage';
+    $scope.brandSmall = 'Dg';
```
Comment (arielron) [inline public/dist/application.js]: Please use applicatiom constants

File: public/dist/application.min.css [hunk 6]
```diff
@@ -13528,6 +13533,34 @@ strong {
 .btn.btn-block {
   margin-bottom: 5px;
 }
+.btn-circle {
+  color: black;
```
Comment (arielron) [inline public/dist/application.min.css:13537]: please order CSS properties alphabetically, here and everywhere.

File: public/modules/home/controllers/header.client.controller.js [hunk 7]
```diff
@@ -4,13 +4,14 @@ angular.module('app.home').controller('HeaderController', ['$window', '$rootScop
   function ($window, $rootScope, $scope, $state, ngProgressFactory, $modal) {
 
     $scope.brand = 'The Digital Garage';
+    $scope.brandSmall = 'Dg';
```
Comment (arielron) [inline public/modules/home/controllers/header.client.controller.js]: use an application constant

**Comment by rleira** [comment]
Code has been updated to fix the issues commented, please notice that files under 'dist' don't need to be reviewed as they are generated from the source.

**Pull request merged**